### DISCUSSION TITLE
CKB 怎么生成地址？

### DISCUSSION THREAD

**Discussion by scrawld** [nervosnetwork/ckb#2418] [General]
CKB 节点有提供生成地址的接口么 就好比 `ckb-cli account new` 命令

**Comment by doitian** [comment]
CKB 不会提供和内置任何钱包相关的功能。请使用最佳实践生成和管理 secp256k1 key pair，通过 https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0021-ckb-address-format/0021-ckb-address-format.md 规范可以生成地址
